pyfr/integrators/dual/pseudo/__init__.py

```diff
@@ -1,12 +1,45 @@
 # -*- coding: utf-8 -*-
 
+from pkg_resources import resource_listdir
 import re
 
 from pyfr.integrators.dual.pseudo.pseudocontrollers import BaseDualPseudoController
-from pyfr.integrators.dual.pseudo.pseudosteppers import BaseDualPseudoStepper
+from pyfr.integrators.dual.pseudo.pseudosteppers import BaseDualPseudoStepper, DualDenseRKPseudoStepper
 from pyfr.integrators.dual.pseudo.multip import DualMultiPIntegrator
 from pyfr.util import subclass_where
 
+# Create subclasses for all tabulated dense schemes
+tabschemes = []
+rpaths = resource_listdir(__name__, 'densepseudoschemes/schemes')
```

Comment:
Have the schemes a level up.  They should just be schemes (with appropriate flags to mark them as being not time accurate).

Reply:
I am just about to push. This is the only point that I would like to keep as it is. I have come up with better names though. My logic is to have schemes in folders

tabulatedpseudosteppers/type/ 

under intergrators/dual/pseudo.

If we want to add any type of pseudosteppers we may just add a new folder and make register_tabulated_pseudo_steppers() to include them.

Similarly for the time-accurate schemes. The stepper containers would be

tabulateschemes/type/

under integrators/std/.

Reply:
I'm not sure why.  A stepper is a stepper.  Sure, some steppers have different properties to others, but that does not change the fact that they are still a stepper.  I am also unsure about putting schemes in folders; any scheme which does the same thing should be together.  (To play devil's advocate: where would the classical RK4 scheme go in this arrangement?  What 'type' is it?  What would go in the folder itself?)  Folders should really only be used where name disambiguation is required.  For instance, in the quadrature rules we have several "gauss-legendre" rules (line, quad, hex).  Thus we need to group first by folder.  But beyond that all schemes which integrate a hex to together, etc, etc.

Reply:
Ok. All schemes are now in one folder. 
How about pseudo-vermeire-s7-po0.txt as a flag?

Reply:
So the name is `pseudo-vermeire`?  That works with me if you guys are happy.  However, be sure that we do not find ourself in an awkward place if you guys develop some more schemes in the future!

For flags I think `s` is good for stages, `p` for (temporal order of accuracy), `sp` for solution order of accuracy (optional).  We may also want to define an `e` flag now for error estimation support.

Reply:
I meant pseudo as a prefix for all schemes which should not be used as a time accurate stepper. The name would be just vermeire.